test_runner/src/main/kotlin/ftl/cli/firebase/test/android/AndroidRunCommand.kt

```diff
@@ -44,9 +44,9 @@ class AndroidRunCommand : CommonRunCommand(), Runnable {
             MockServer.start()
         }
 
-        val config = AndroidArgs.load(Paths.get(configPath), cli = this).validate()
+        val config = AndroidArgs.load(Paths.get(configPath), cli = this).validate().copy(obfuscateDumpShards = obfuscate)
```

Comment:
If you have a better idea to pass the obfuscate parameter to newTestRun tell me please